Week-2/HTML-ADVANCED/RESTUARANT-REVIEWS/greekHouse.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+<header>
+    <h1>Restaurant Reviews</h1>
+    <nav>
+        <a href="">Home</a>
+        <a href="">Add Review</a>
+        <a href="">About</a>
+    </nav>
+</header>
+<main>
+    <section>
+        <h2>Greek House</h2>
+        <img src= "" alt= "Greek house"> <br>
```

Comment:
The `src `attribute for images is currently empty. Make sure to provide a valid path to the image files, and use descriptive `alt `text for accessibility.

`<img src="path-to-greek-house.jpg" alt="Interior of Greek House restaurant">
`